HandEyeCalibration/UtilizeEyeInHandCalibration/UtilizeEyeInHandCalibration.cpp

```diff
@@ -0,0 +1,83 @@
+/*
+Utilize the result of eye-in-hand calibration to transform (picking) point
+coordinates from the camera frame to the robot base frame.
+*/
+
+#include <Eigen/Core>
+#include <cmath>
+
+#include <iostream>
+#include <opencv2/core/core.hpp>
+
+Eigen::MatrixXd cv2eigen(const cv::Mat &cv_mat)
+{
+    if(cv_mat.dims > 2)
```

Comment:
I don't know OpenCV. Is it possible to do this check statically instead? Does OpenCV have a matrix type which is guaranteed to be 2D?

Reply:
@chrisasc do you know?

Reply:
I'm not able to find a specific 2D matrix, sorry :(